src/pages/ReimbursementAccount/ReimbursementAccountPage.tsx

```diff
@@ -395,7 +400,23 @@ function ReimbursementAccountPage({route, policy, isLoadingPolicy}: Reimbursemen
     } else if (throttledDate) {
         errorText = translate('bankAccount.hasBeenThrottledError');
     } else if (hasUnsupportedCurrency) {
-        errorText = translate('bankAccount.hasCurrencyError');
+        errorText = (
+            <Text style={styles.flexRow}>
+                {translate('bankAccount.hasCurrencyError.phrase1')}
+                <TextLink
+                    style={styles.link}
+                    onPress={() => {
+                        const routeToNavigate = isSmallScreenWidth
+                            ? ROUTES.WORKSPACE_OVERVIEW.getRoute(policyIDParam, Navigation.getActiveRoute())
+                            : ROUTES.WORKSPACE_INITIAL.getRoute(policyIDParam, Navigation.getActiveRoute());
+                        Navigation.goBack(routeToNavigate);
+                    }}
+                >
+                    {translate('bankAccount.hasCurrencyError.link')}
+                </TextLink>
+                {translate('bankAccount.hasCurrencyError.phrase2')}
+            </Text>
+        );
```

Comment:
The way this is done right now conflicts with our guidance for how translation functions should work. 
https://github.com/Expensify/App/blob/75bdb4c8c62898f06f2a32d01afb68d35296e4c2/README.md#L744-L748

Instead, can we please include the link as html in the translated message? In [this PR](https://github.com/Expensify/App/pull/58099/files#diff-4935ec01feca8054d8a080f44501a37a1cf1398ffd19e0b46d3551a56e5ba313) I've added the ability for errors to render links, so maybe we can hold until that's merged?